Sources/TensorFlow/Core/TensorGroup.swift

```diff
@@ -216,10 +262,29 @@ extension Array: TensorArrayProtocol where Element: TensorGroup {
             count: Int(count)).joined())
     }
 
+    public var _tensorHandles: ([_AnyTensorHandle]) {
+        var result: [_AnyTensorHandle] = []
+        result.reserveCapacity(Int(self._tensorHandleCount))
+        for elem in self {
+            result += elem._tensorHandles
+        }
+        return result
+    }
+
     public init(_owning tensorHandles: UnsafePointer<CTensorHandle>?, count: Int) {
         let size = count / Int(Element._tensorHandleCount)
         self = Array((0..<size).map { Element.init(
             _owning: tensorHandles?.advanced(by: $0 * Int(Element._tensorHandleCount)))
         })
     }
+
+    public init(handles: [_AnyTensorHandle]) {
+        let size = handles.count / Int(Element._tensorHandleCount)
+        self = Array((0..<size).map {
```

Comment:
The `map(_:)` method already produces an array. No need to call the array initializer.

```suggestion
        self = (0..<size).map {
```

Reply:
Done.